src/API/Surface.hs

```diff
@@ -83,17 +78,6 @@ loadTheory config isTPTP = case configInput config of
             Right (Input thy dps) -> do
               let config' = config {configSkolemDepth = dps}
               return $ Right (config', thy)
-  --where
```

Comment:
I'm happy to see all this activity on Razor.
Today was a travel day for me but I plan to dive in tomorrow.
Especially concerning non-pure mode, a question that seems crucail to me.

Dan

On Sat, Jun 18, 2016 at 10:54 PM, Salman Saghafi <notifications@github.com>
wrote:

> In src/API/Surface.hs
> <https://github.com/salmans/Razor/pull/78#discussion_r67609482>:
>
> > @@ -83,17 +78,6 @@ loadTheory config isTPTP = case configInput config of
> >              Right (Input thy dps) -> do
> >                let config' = config {configSkolemDepth = dps}
> >                return $ Right (config', thy)
> > -  --where
>
> I don't remember why this was committed. Eventually, we need to be able to
> load TPTP files.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/salmans/Razor/pull/78/files/c879e8a9e619d1a061ee5a566dbdc12c119940cb#r67609482>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAxJeaNXi6BCJ5QUhMLVFAIxhoN4-Mjvks5qNK9PgaJpZM4I5Eib>
> .
>



-- 
Dan Dougherty
Professor of Computer Science
Ombuds
Worcester Polytechnic Institute